xarray/core/rolling.py

```diff
@@ -54,17 +54,19 @@ def __init__(self, obj, windows, min_periods=None, center=False, keep_attrs=None
         ----------
         obj : Dataset or DataArray
             Object to window.
-        windows : A mapping from a dimension name to window size
+        windows : mapping of hashable to int
+            A mapping from a dimension name to window size
+
             dim : str
```

Comment:
I think we can remove `dim` and `window`. It renders weirdly: https://xray--4286.org.readthedocs.build/en/4286/generated/xarray.core.rolling.DataArrayRolling.html